test/test_models.py

```diff
@@ -6,44 +6,43 @@
 from delirium.dns.cache import get_addr_range
 from delirium.dns.cache import CacheDatabase
 
+@pytest.fixture()
+def test_db():
+    yield CacheDatabase(DEFAULT_ADDR_RANGE, DEFAULT_CACHE_DURATION, DEFAULT_DB_PATH)
 
+@pytest.mark.usefixtures('test_db')
 class TestCacheDatabase():
```

Comment:
Let's rename this to `TestDefaultCache`.  This makes it a little more explicit that we're going to be testing a default instance of `CacheDatabase` in these tests.